### PR TITLE
[dev] No PBI - Added Button Types And Fixed Docs' Button API Page.

### PR DESCRIPTION
**Pull request by cbrewer1689**

### CODE REVIEW

**Comment by cbrewer1689** [comment]
Depends on #379 so we can complete the Button's types to use TS enums.

**Review by groberts314 — APPROVED** [review]
LGTM 👍 :shipit: 🚀

File: src/inputs/button/button.jsx [hunk 1]
```diff
@@ -2,25 +2,172 @@ import {
...
+     */
+    title: PropTypes.string,
+    /**
+     * Set transparent styles.
+     */
+    transparent: PropTypes.bool,
+    /**
+     * Set transparent styles.
+     */
+    type: PropTypes.oneOf(Object.values(Type)),
```
Comment (groberts314) [inline src/inputs/button/button.jsx:132]: Let's fix up this documentation comment (which appears to be copied and pasted) and maybe link to that MDN page about browser's button attribute.

File: src/inputs/button/models/type.model.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+enum Type {
```
Comment (groberts314) [inline src/inputs/button/models/type.model.ts:1]: Let's rename to `ButtonType` oer our discussion

Comment (groberts314) [reply src/inputs/button/models/type.model.ts]: Type renamed in 37f08fd.  Resolving,

File: src/inputs/button/button.jsx (hunk 1 above)
Comment (groberts314) [reply src/inputs/button/button.jsx]: Resolved in a9ef7f5.

**Review by groberts314 — APPROVED** [review]
Approved latest changes including use of TS enums for Button props.  Love it!

**Pull request merged**